### PR TITLE
User can create a todo.

### PR DESCRIPTION
**Pull request by riley-rangel**
cd-issue-9 0

### CODE REVIEW

File: server/gateway.js [hunk 1]
```diff
@@ -12,8 +12,13 @@ module.exports = function createGateway(collection) {
...
     async create(newTodo) {
       const todo = Object.assign({ id: uuidv4() }, newTodo)
-      const created = await collection.insertOne(todo)
-      return created.result
+      try {
+        await collection.insertOne(todo)
+        return todo
+      }
+      catch (error) {
+        return error
```
Comment (thebearingedge) [inline server/gateway.js]: You can `omit` this `try/catch` block. If this method throws, it'll be caught in your `POST` handler.

File: nodemon.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "ignore": [
+    "server/public"
+    ]
```
Comment (thebearingedge) [inline nodemon.json]: whitespace

File: test/index.test.js [hunk 3]
```diff
@@ -17,10 +17,10 @@ describe('app', () => {
     server.close(() => done())
   })
 
-  describe('GET: "/"', () => {
+  describe('GET: "/api/"', () => {
 
     it('sends an object with repo name, description, and link', done => {
-      request('http://localhost:3000/', (error, response, body) => {
+      request('http://localhost:3000/api/', (error, response, body) => {
```
Comment (thebearingedge) [inline test/index.test.js]: Use your `process.env.PORT` instead of hard-coding it.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: test/index.test.js (hunk 3 above)
Comment (riley-rangel) [reply test/index.test.js]: Oops, didn't even realize I did that.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**